psql-backups-iam-auth.sh

```diff
@@ -95,8 +95,23 @@ aws configure set s3.signature_version s3v4
 
 # Install the postgres tools matching the engine version
 get_time_now
-echo "$time_now Postgres dump. installing dependencies..."
-sudo amazon-linux-extras install -y postgresql$PSQL_TOOLS_VERSION > /dev/null
+echo "$time_now Postgres dump. installing dependencies for postgresql$PSQL_TOOLS_VERSION ..."
+
+if [[ $majorVersion -ge 12 ]]; then
+  # amazon-linux-2 doesn't have postgresql packages above V11.
+  sudo tee /etc/yum.repos.d/pgdg.repo<<EOF
+[pgdg$PSQL_TOOLS_VERSION]
+name=PostgreSQL $PSQL_TOOLS_VERSION for RHEL/CentOS 7 - x86_64
+baseurl=https://download.postgresql.org/pub/repos/yum/$PSQL_TOOLS_VERSION/redhat/rhel-7-x86_64
+enabled=1
+gpgcheck=0
+EOF
+  sudo yum makecache
+  sudo yum install postgresql$PSQL_TOOLS_VERSION 
```

Comment:
```suggestion
  sudo yum install "postgresql${PSQL_TOOLS_VERSION}"
```

Reply:
Mixing variables and literals in bash is a little ugly, using this format makes it easier to read IMO

Reply:
Makes sense. Should I do the same thing in the file creation above?

Reply:
In a HEREDOC, I think it's a little less important, but wouldn't hurt to change as well